Notifier-service/src/services/streams/conferenceStream.ts

```diff
@@ -39,16 +43,72 @@ export class ConferenceStreamMongo extends ConferenceStream {
             })
         })
             .pipe(
-                share()
+                filter()
```

Comment:
* keep the share() , share is used to convert Observable , into subject. Observable restarts everytime a listener is attached. share() is used to turn into a subject. 
* filter need not replace it. Also filter requires a argument callback function.

Reply:
I kept the filter empty as i want to know, i want to apply filter on which and what am i supposed to get from the filter?

Reply:
* You will use filter only when u r using getStream() to create a new filtered observer/subject from the primary one already created. The getStream emits every event. The filter can have the function to filter based on event type i.e update , create etc.
* Or you can create a new watch with pipe and avoid using filter. In this case we will have 3 extra independent subjects.